BombsExcersice/BombsExcersice.cpp

```diff
@@ -1,10 +1,173 @@
-// BombsExcersice.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
 #include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+
+enum Type {Empty, Bomb, Wall};
+
+class Cell
+{
+public:
+    Cell() : type{ Empty }, lifetime{ 0 }, maxLifetime{ 3 } {}
+
+    void setType(Type t) { type = t; }
+    Type getType() const { return type; }
+
+    void becomeEmpty()
+    {
+        type = Empty;
+        lifetime = 0;
+    }
+    
+    int getLifetime() const { return lifetime; }
+    int getMaxLifetime() const { return maxLifetime; }
+    void increaseLifetime() { ++lifetime; }
```

Comment:
The more clear function name would be `incrementLifeTime()`, this shows on which amount of time you increase it.